rest/example/main.go

```diff
@@ -12,6 +13,7 @@ import (
 func main() {
 	getAllTickersSnapshot()
 	listTrades()
+	getAggregateBarsLaunchpad()
```

Comment:
I don't like having this example in this file since it's not the main use case. I was thinking of creating a separate `main.go` for it but wasn't sure where to put it. Could I make a subdir and keep the existing example in `rest/example/common/main.go` and move the new one to `rest/example/launchpad/main.go`?

Reply:
Another option is you could add an arg that let's people choose which example to run, e.g. `POLYGON_API_KEY=... go run rest/example/main.go launchpad`. I'm fine with separate examples too though but we'll need to make sure we update any broken links

Reply:
I like that. I'll make that change